docs/conf.py

```diff
@@ -94,6 +94,7 @@
     "source_repository": "https://github.com/pradyunsg/furo/",
     "source_branch": "main",
     "source_directory": "docs/",
+    "source_edit_link": "",
```

Comment:
If you could remove this line, this PR is good to go!

Reply:
```suggestion
```

Reply:
Done. Many thanks for getting around to this :)